packages/amplication-git-utils/src/git/github.service.ts

```diff
@@ -350,4 +352,33 @@ export class GithubService {
       })
     ).token;
   }
+
+  async getRepository(
```

Comment:
Why do we need this function? I assume we already have it somewhere

Reply:
Please have a look, I didn't find any